projects/briey/memtest/src/main.c

```diff
@@ -1,19 +1,20 @@
 #include <stdint.h>
 #include <stdlib.h>
+#include <stdio.h>
 #include <briey.h>
 
-#define PATTERN       (0b10101010u)
+#define PATTERN       (0x55555555U) // Repeating 0b0101...
 
 #define UART_DATA_LEN (8U)
 #define UART_BAUD     (115200U)
 
 extern void flushDataCache(); // From /vga/src/crt.S
-extern int _heap_start, _heap_end;
-volatile uint8_t state = 0;
+extern char _heap_start, _heap_end, _memTree_start, _memTree_end;
```

Comment:
Why the change to char?

Reply:
To follow example at the bottom of: https://sourceware.org/binutils/docs/ld/Source-Code-Reference.html#Source-Code-Reference